prog/vm/host_nexus.rb

```diff
@@ -245,8 +245,16 @@ def get_boot_id
   end
 
   def available?
-    sshable.cmd("true")
-    true
+    ssh_session = sshable.start_fresh_session
+    is_available = false
+
+    begin
+      vm_host.perform_health_checks(ssh_session)
+    ensure
+      ssh_session&.close
```

Comment:
```suggestion
      ssh_session.close
```

Unless `sshable.start_fresh_session` can return nil, by the time you get here, it should be set.